default.vcl

```diff
@@ -23,10 +23,10 @@ sub vcl_recv {
     elif (req.url ~ "^\/content\/.*$") {
         set req.backend_hint = dynBackend.backend("content-public-read");
     }
-    elif (req.url ~ "^\/enrichedcontent\/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$") {
+    elif (req.url ~ "^\/enrichedcontent\/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}\??.*$") {
```

Comment:
I think these regexes could be further simplified by removing the optional part...